src/components/modal/Modal.scss

```diff
@@ -144,8 +144,23 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: fixed;
+            top: 0;
+            left: 0;
+            z-index: 2;
+            display: flex;
```

Comment:
Since `.modal-header-container` only has one child, do we need this style?

Reply:
If we're using the `display: flex` to vertically align the `.modal-header`, we should add `align-items: center` and remove the top padding for `.modal-header`.

This will make it more maintainable than keeping the padding of `.modal-header` in sync with the height of `.modal-header-container`.

Reply:
Added `align-items: center` to `.modal-header-container` and also removed top padding from `.modal-header`.